account/accounts.go

```diff
@@ -245,6 +248,27 @@ func (m *Manager) SelectAccount(walletAddress, chatAddress, password string) err
 	return nil
 }
 
+// InjectChatAccount initializes selectedChatAccount with privKey
```

Comment:
Maybe `SetChatAccount`? "Inject" suggests that it is added permanently or to some storage but it just sets `m.selectedChatAccount`.